spring/src/test/java/org/axonframework/spring/config/AutoWiredEventSourcedAggregateTest.java

```diff
@@ -16,6 +16,7 @@
 
 package org.axonframework.spring.config;
 
+import jakarta.persistence.Id;
```

Comment:
I assume there's no equivalent of this in the `hibernate-6-integrationtests` module, as that module doesn't include Spring, right?

Reply:
I didn't add changes to the `SpringAxonAutoConfigurer` for it to work with jakarta, since that is already deprecated. however it would be easy to add the changes and also test in `spring-boot-3-integrationtest`? Not sure what is wise here.

Reply:
Yes, and since the test doesn't integrate with hibernate.